nginx.conf

```diff
@@ -0,0 +1,81 @@
+events {
+    worker_connections 1024;
+}
+
+# frontend
+http {
+    server {
+        listen 80;
+        server_name sim.genlayer.com;
+        return 301 https://$server_name$request_uri;
+    }
+
+    server {
+        listen 443 ssl;
+        http2 on;
+        server_name sim.genlayer.com;
```

Comment:
the `server_name` should be configurable (probably through environment variables) on all of these `server`s, since it's something that will very likely change on each deployment.

Reply:
I've added `SERVER_NAME` env var